src/resources/elements/optionfield.js

```diff
@@ -189,6 +189,33 @@ export class Optionfield extends Field {
   }
 
   /**
+   * Get the label(s) of the currently chosen option(s)
+   * @return {String|String[]} The label or labels depending on the format of
+   *                           this Optionfield.
+   */
+  getChoiceLabel() {
+    if (this.format === 'dropdown') {
```

Comment:
A comment for such a simple `if` if doesn't really increase readability at all in my opinion.